app/src/main/java/com/locked/shingranicommunity/event/EventActivity.kt

```diff
@@ -26,4 +31,13 @@ class EventActivity : FragmentActivity(), EventComponentProvider {
         }
         return super.onOptionsItemSelected(item)
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (data != null){
+            if (requestCode == Constant_Utils.ONE_03){
+                supportFragmentManager.findFragmentById(R.id.fragment)?.onActivityResult(requestCode,resultCode,data)
+            }
+        }
+    }
```

Comment:
change to:
```kotlin
    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        super.onActivityResult(requestCode, resultCode, data)
        supportFragmentManager.findFragmentById(R.id.fragment)?.onActivityResult(requestCode,resultCode,data)
    }
```

then you'll use `if (requestCode == Constant_Utils.ONE_03)` inside the fragment.